modules/mgmt-creating-a-node-maintenance-cr.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// virt/node_maintenance/mgmt-node-maintenance.adoc
+
+[id="mgmt-creating-a-node-maintenance-cr_{context}"]
+= Setting a node to maintenance mode with a custom resource
+
+You can put a node into maintenance mode with a custom resource. When you apply a node maintenance CR, all allowed pods are evicted and the node is shut down. Evicted pods are queued to be moved to another node in the cluster.
+
+.Prerequisites
+
+* Install the {product-title} CLI (oc).
+* Log in to the cluster as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create the following node maintenance schedule CR, and save the file as `nodemaintenance-cr.yaml`:
++
+[source,yaml]
+----
+apiVersion: nodemaintenance.kubevirt.io/v1beta1
+kind: NodeMaintenance
+metadata:
+  name: maintenance-example  <1>
+spec:
+  nodeName: node-1.example.com <2>
+  reason: "Node maintenance" <3>
+----
+<1> Node maintenance CR name
+<2> The name of the node to be put into maintenance mode
+<3> Plain text description of the reason for maintenance
++
+.  Apply the node maintenance schedule by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f nodemaintenance-cr.yaml
+----
+
+. Check the progress of the maintenance schedule by running the following command, replacing `<node-name>` with the name of your node:
++
+[source,terminal]
+----
+$ oc describe node <node-name>
+----
++
+You should see output similar to the below:
++
+[source,terminal]
+----
+Events:
+  Type     Reason                     Age                   From     Message
+  ----     ------                     ----                  ----     -------
+  Normal   NodeNotSchedulable         61m                   kubelet  Node node-1.example.com status is now: NodeNotSchedulable
+----
++
+See more information about current maintenance schedules by running the following command:
```

Comment:
This would almost be worth having as a separate module in itself.
However if it's to stay here, it would be better under a `.Verification steps` heading. ie
```
.Verification steps

* View the current maintenance schedules:
+
[source...
```

Reply:
I created a seperate module for this like you suggested. Probably best.